libgraphics/include/pivot/graphics/interface/IPipelineBuilder.hxx

```diff
@@ -10,9 +10,11 @@ namespace pivot::graphics::interface
 class IPipelineBuilder
 {
 public:
-    /// Set the pipeline layout, can be chained
+    /// @brief Get the debug name of the builders
+    virtual constexpr const char *getDebugPipelineName() const noexcept = 0;
```

Comment:
`virtual constexpr` ? Are you sure it makes sense to add constexpr ? :thinking:

Reply:
It is a constant expression which return a static constant string, and it is pure (so virtual)